tab3_new.py

```diff
@@ -0,0 +1,339 @@
+
+from PyQt5.QtWidgets import *
+from PyQt5.QtGui import *
+from PyQt5.QtCore import *
+from triggerBoard.py import *
+import sys
+import logging
+import subprocess
+import os
+import signal
+import time
+import glob
+import select
+import os.path
+
+'''
+#use for debug
+def runClass(num):
+	x= num
+'''
+
+class trigger_board_tab(QWidget):
+	def __init__(self):
+		super().__init__()
+		
+		#set up all the buttons and textbox in this tab
+		self.setTitleLabel()
+		self.setFirmwareBtn()
+		self.setCoincidenceTimeBtn()
+		self.setHistogramBtn()
+		self.setToggleOutputBtn()
+		self.setToggleClockInputBtn()
+		self.setAdjustClockPhaseBtn()
+		self.setRandomSeedBtn()
+		self.setPrescaleBtn()
+		self.setActiveClockBtn()
+		self.setTogglePhaseBtn()
+		self.setSendHistogramBtn()
+		self.setDeadTimeBtn()
+		self.setAdjustClock1Btn()
+		self.setToggleTriggerRollingBtn()
+		self.setInputTriggerMaskBtn()
+		self.setTriggerBtn()
+		self.setClockCyclesBtn()
+		self.setReClockBtn()
+	
+		
+		self.show()
+		
+	#Set the tile of this tab
+	def setTitleLabel(self):
+		self.label = QLabel(self)
+		self.label.setText("Control Trigger Board")
+		self.label.setFont(QFont("Arial",30))
+		self.label.move(300,20)
+		
+	#output firmware version
+	def setFirmwareBtn(self):
+		self.btn = QPushButton("Firmware Version",self)
```

Comment:
Name these buttons more clearly than as btn, btn2, etc. The names should represent what they do